_delphi_utils_python/delphi_utils/geomap.py

```diff
@@ -416,6 +421,15 @@ def replace_geocode(
         from_col = from_code if from_col is None else from_col
         new_col = new_code if new_col is None else new_col
 
+        if from_code == "fips" and pop_col:
+            # Zero out the duplicate population in megafips to avoid double counting.
+            megafips_to_zero = [i in self._megafips_to_zero(df[from_col]) for i in df[from_col]]
```

Comment:
does Python do the smart thing here and compute `self._megafips_to_zero(df[from_col])` once, or will it run it N times?

Reply:
probably not, ill extract it to a variable